pkgs/by-name/me/melonDS/package.nix

```
@@ -3,6 +3,7 @@
   SDL2,
   cmake,
   extra-cmake-modules,
+  enet,
```

```suggestion
   enet,
   extra-cmake-modules,
```

ascii-betical ordering